services/ui-src/src/page/DetailView.tsx

```diff
@@ -78,7 +80,7 @@ const DetailView: React.FC<{ pageConfig: OneMACDetail }> = ({ pageConfig }) => {
   }
   const loadDetail = useCallback(
     async (ctrlr?: AbortController) => {
-      let fetchedDetail;
+      let fetchedDetail: ComponentDetail | undefined;
```

Comment:
I think this is correct, because the async nature of the fetchedDetail means that the page could do all sorts of things while it is still undefined...